bindings/java/src/test/java/org/openucx/jucx/UcpEndpointTest.java

```diff
@@ -735,7 +735,8 @@ public void onSuccess(UcpRequest request) {
             sendData.getMemory().getAddress(), 2L, UcpConstants.UCP_AM_FLAG_PERSISTENT_DATA, null,
             new UcpRequestParams().setMemoryType(memType).setMemoryHandle(sendData.getMemory()));
 
-        while (!Arrays.stream(requests).allMatch(r -> (r != null) && r.isCompleted())) {
+        while (!Arrays.stream(requests).allMatch(r -> (r != null) && r.isCompleted()) ||
+             (persistantAmData.get() == null)) {
```

Comment:
If we are already touching this code, maybe fix the spelling mistake?
`persistant` -> `persistent`

Reply:
next PR